playground/static/main.js

```diff
@@ -223,7 +245,7 @@ function updateOutput(input) {
                 render.style.display = "none";
 
                 debugEditor.session.setMode("");
-                debugEditor.setValue(ast_debug(input));
+                debugEditor.setValue(await compilerAction({ type: "ast", source: input }));
```

Comment:
This is probably wrong, it seems strange to do the action type `ast` on both `ast-debug` and `ast` arms, probably a typo. See line 239.